roles/openshift_master_facts/tasks/main.yml

```diff
@@ -77,6 +77,9 @@
       kube_admission_plugin_config: "{{openshift_master_kube_admission_plugin_config | default(None) }}"  # deprecated, merged with admission_plugin_config
       oauth_always_show_provider_selection: "{{ openshift_master_oauth_always_show_provider_selection | default(None) }}"
       image_policy_config: "{{ openshift_master_image_policy_config | default(None) }}"
+      image_policy_allowed_registries_for_import: "{{ openshift_master_image_policy_allowed_registries_for_import | default(None) }}"
+      image_policy_additional_allowed_registries_for_import: "{{ openshift_master_image_policy_additional_allowed_registries_for_import | default(None) }}"
+      default_image_policy_allowed_registries_for_import: "{{ default_image_policy_allowed_registries_for_import }}"
```

Comment:
Instead of  setting the defaults here, is it possible to load them from `set_allowed_registries()`. Not sure it's the best way but the current approach is not consistent with anything around.

Reply:
Resolved. By default, everything is allowed.